pkg/kev/config/extensions_test.go

```diff
@@ -105,38 +101,55 @@ var _ = Describe("Extentions", func() {
 					}
 				})
 
+				JustBeforeEach(func() {
+					svc.Extensions = extensions
+				})
+
 				When("workload is invalid", func() {
-					It("returns an error", func() {
-						parsedCfg, err = config.K8SCfgFromMap(extensions)
-						Expect(err).To(HaveOccurred())
-						Expect(err.Error()).To(ContainSubstring("Workload.Replicas is required"))
+					It("it is ignored and returns defaults", func() {
+						parsedCfg, err = config.K8SCfgFromCompose(&svc)
```

Comment:
Similar comment here to the one above. This could be moved to the `JustBeforeEach` block. Also I think it'd be good to set expectation on `parsefCfg` here?

Reply:
fixed

Reply:
can't fine expectation on `parsedCfg` in this block. I think we should have one as we expect defaults here?